src/svgs/logo-inline.svg

```diff
@@ -0,0 +1,5 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 971.997 180.498">
+  <title>Ethos Logo</title>
```

Comment:
It looks like we lose the title in this branch:

**Current:**
![Screen Shot 2019-12-02 at 12 43 16 PM](https://user-images.githubusercontent.com/11511539/69993820-7298f980-1501-11ea-9765-51dbdd6176ea.png)

**This branch**
![Screen Shot 2019-12-02 at 12 43 31 PM](https://user-images.githubusercontent.com/11511539/69993819-7298f980-1501-11ea-8fd0-ca9e4b989c63.png)